scripts/forking/template/whitelabel/opcodes.py

```diff
@@ -0,0 +1,18 @@
+import copy
+
+from eth_utils.toolz import merge
+
+
+from eth.vm.forks.petersburg.opcodes import (
+    PETERSBURG_OPCODES,
+)
+
+
+UPDATED_OPCODES = {
```

Comment:
mypy wants this to be:
```suggestion
UPDATED_OPCODES: Dict[int, eth.vm.opcode.Opcode] = {
```